### ISSUE TITLE
Earth Day Projects - Tweet Thread - April 14

### ISSUE THREAD

**Issue by c0ric0ri** [Giveth/giveth-planning#1317]
Create a summary tweet of each project from our Earth Day campaign, so we can post one a day starting the end of this week and going through Earth Day.

We'll also need an intro tweet for a April 22 "thread of threads" linking to each of these individual tweets.

**Comment by c0ric0ri** [comment]
@danelsuga would you be up for drafting this?

**Comment by c0ric0ri** [comment]
Started a draft here: https://docs.google.com/document/d/18hZP_tvnT7qOQ1BsoMLJKXLICIaNZm8hMxF9UcQx7LY/edit

**Comment by laurenluz** [comment]
ugh google docs is not working for me these days to I can't open the doc. Here's the text:

**Comment by laurenluz** [comment]
And FYI - it's worthwhile copy/pasting project text into chat GPT - training it with some tweets & then getting it to summarize the project descriptions into tweets we can use! Then you just need to find the right twitter handle to tag the project and add in the bit.ly link 😄 

This was all part of my presentation today, and it really speeds things up.

**Comment by c0ric0ri** [comment]
Ok good tip @laurenluz ! Will add more tomorrow trying chat GPT!

**Comment by c0ric0ri** [comment]
@laurenluz drafted the thread of threads at the bottom of the doc to put out for Earth Day tomorrow.

**Comment by laurenluz** [comment]
@c0ric0ri Chat GPT is great at producing 1st drafts, but the content needs human review. A lot of these tweets were weird, wordy or vague... and seemed like they were created with AI and without proper QA.

It took me 30 min to get through it and try to make it understandable for me. I got it a good enough point imo and scheduled now for tomorrow.

**Comment by c0ric0ri** [comment]
@laurenluz I didn't use Chat GPT to write these tweets. They were not crafted with AI.

I went through each project page and their updates and used content from there.

I was trying to find new content to write above already existing descriptions of the projects. Most of them haven't posted updates this year, so I pulled from other angles and thought it was fine - I agree some sentences could be a bit stronger, but I wasn't expecting it to create a lot of extra work for you. But I've been feeling a bit burned out today and was trying to get this draft done fast so it wouldn't be late for you to schedule before the weekend, so maybe I'm off my game today.
